packages/template-retail-react-app/app/pages/product-detail/index.test.js

```diff
@@ -123,7 +120,7 @@ describe('product set', () => {
             // Seems like rendering the modal takes a bit more time
             {timeout: 5000}
         )
-    })
+    }, 30000)
```

Comment:
That seems like a _big_ jump for the timeout! Is there any better way to address it?

Reply:
That's big timeout, yes.. but it's not the first time we've done it in the project ;)

On the other hand, is there a better way? Well, Kevin mentioned this last time: investigate why modal is rendering slowly during testing, especially in CI environment. It's something that we haven't had a chance to do, but we can revisit it for sure.